GetCertificate/__tests__/printer_vaccine_detail_de.test.ts

```diff
@@ -43,4 +21,33 @@ describe(`Printer - Test - Detail - ${language}`, () => {
     const result = printDetails(some(language), certificate);
     expect(result).toMatchSnapshot();
   });
+
+  it.each`
+    doseNumber | totalDoses
+    ${3}       | ${2}
+    ${3}       | ${3}
+    ${2}       | ${1}
+  `(
+    "should print markdown in case of booster $doseNumber/$totalDoses",
+    ({ doseNumber, totalDoses }) => {
+      const certificate = pipe(
+        Certificates.decode({
+          ...aValidVaccinationCertificate,
+          v: [
+            {
+              ...aValidVaccinationCertificate.v[0],
+              dn: doseNumber,
+              sd: totalDoses
+            }
+          ]
+        }),
+        getOrElseW(_ => {
+          throw "Error decoding object";
+        })
+      );
+
+      const result = printDetails(some(language), certificate);
+      expect(result).toMatchSnapshot();
+    }
+  );
```

Comment:
Could we put it in a tests_utils and reuse it for all lenguages?

Reply:
I wanted to do this in a dedicated PR, to avoid confusion in this one